components/meta.js

```diff
@@ -10,6 +10,9 @@ import Fonts from "./styles/fonts";
 const { publicRuntimeConfig } = getConfig();
 const SITE_NAME = publicRuntimeConfig.SITE_NAME || "api.gouv.fr";
 const SITE_URL = publicRuntimeConfig.SITE_URL || "https://api.gouv.fr";
+const SITE_DESCRIPTION =
+  publicRuntimeConfig.SITE_DESCRIPTION ||
+  "Le site qui simplifie le partage et la circulation des données administratives";
```

Comment:
can you keep "Simplifiez ..." instead of "Le site qui simplifie ...". This is a change that we made with @patama , and we want to keep this for now.